packages/lex-cli/src/codegen/client.ts

```diff
@@ -306,7 +306,7 @@ function genRecordCls(file: SourceFile, schema: RecordSchema) {
     const method = cls.addMethod({
       isAsync: true,
       name: 'list',
-      returnType: `Promise<{records: ({uri: string, value: ${typeModule}.Record})[]}>`,
+      returnType: `Promise<{cursor?: string, records: ({uri: string, value: ${typeModule}.Record})[]}>`,
```

Comment:
This change was needed so that the convenience `list()` methods (e.g. `client.app.bsky.post.list()`) supported the cursor output.